StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableLoadingView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  InstitutionTableLoadingView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 1/24/24.
+//
+
+import Foundation
+import UIKit
+@_spi(STP) import StripeUICore
+
+final class InstitutionTableLoadingView: UIView {
+    
+    init() {
+        super.init(frame: UIScreen.main.bounds)
+        let verticalStackView = UIStackView(
+            arrangedSubviews: (0..<10).map({ _ in
+                ShimmeringInstitutionRowView()
+            })
+        )
+        verticalStackView.axis = .vertical
+        verticalStackView.spacing = 0 // the rows have spacing through padding
+        addSubview(verticalStackView)
+        verticalStackView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            verticalStackView.topAnchor.constraint(equalTo: topAnchor),
+            verticalStackView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            verticalStackView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            // height will be automatically determined by the UIStackView and it will be clipped beyond the frame
+        ])
+        clipsToBounds = true
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private class ShimmeringInstitutionRowView: ShimmeringView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .clear
+        
+        let iconView = UIView()
+        iconView.backgroundColor = .backgroundOffset
+        iconView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            iconView.heightAnchor.constraint(equalToConstant: 56),
+            iconView.widthAnchor.constraint(equalToConstant: 56),
+        ])
```

Comment:
rm? `iconView` is never used (looks like this was moved in to the CreateRowIconView helper and just missed being cleaned up?)

Reply:
yes, great catch, thank you!